diff --git a/dist/components/pagination/item.cjs b/dist/components/pagination/item.cjs
index 0a424e533ab3f0315a17a75fdfdbeea9cf7e1bd7..b64f11b25b400d94a757d515ce82d61ecd42eca8 100644
--- a/dist/components/pagination/item.cjs
+++ b/dist/components/pagination/item.cjs
@@ -6,6 +6,8 @@ var React = require('react');
 var iconButton = require('../icon-button/icon-button.cjs');
 var keyboardArrowLeft = require('../icon/icons/hardware/keyboard-arrow-left.cjs');
 var keyboardArrowRight = require('../icon/icons/hardware/keyboard-arrow-right.cjs');
+var keyboardDoubleArrowLeft = require('../icon/icons/hardware/keyboard-double-arrow-left.cjs');
+var keyboardDoubleArrowRight = require('../icon/icons/hardware/keyboard-double-arrow-right.cjs');
 var moreHoriz = require('../icon/icons/navigation/more-horiz.cjs');
 
 var Item = function Item(_ref) {
@@ -61,7 +63,7 @@ var Item = function Item(_ref) {
       size: "small"
     }, /*#__PURE__*/React.createElement(moreHoriz.default, {
       role: "separator"
-    }), /*#__PURE__*/React.createElement(isMorePrev ? KeyboardDoubleArrowLeft : KeyboardDoubleArrowRight, {
+    }), /*#__PURE__*/React.createElement(isMorePrev ? keyboardDoubleArrowLeft.default : keyboardDoubleArrowRight.default, {
       role: "button"
     }));
   }
diff --git a/dist/components/pagination/item.mjs b/dist/components/pagination/item.mjs
index 20ab675e1fd5f6b723e139a453fbf3dfe16cfb4b..0ce606391eb11ede4ef3608934e3da6a475dd61e 100644
--- a/dist/components/pagination/item.mjs
+++ b/dist/components/pagination/item.mjs
@@ -2,6 +2,8 @@ import React, { createElement } from 'react';
 import IconButton from '../icon-button/icon-button.mjs';
 import IconKeyboardArrowLeft from '../icon/icons/hardware/keyboard-arrow-left.mjs';
 import IconKeyboardArrowRight from '../icon/icons/hardware/keyboard-arrow-right.mjs';
+import IconKeyboardDoubleArrowLeft from '../icon/icons/hardware/keyboard-double-arrow-left.mjs';
+import IconKeyboardDoubleArrowRight from '../icon/icons/hardware/keyboard-double-arrow-right.mjs';
 import IconMoreHoriz from '../icon/icons/navigation/more-horiz.mjs';
 
 var Item = function Item(_ref) {
@@ -57,7 +59,7 @@ var Item = function Item(_ref) {
       size: "small"
     }, /*#__PURE__*/React.createElement(IconMoreHoriz, {
       role: "separator"
-    }), /*#__PURE__*/createElement(isMorePrev ? KeyboardDoubleArrowLeft : KeyboardDoubleArrowRight, {
+    }), /*#__PURE__*/createElement(isMorePrev ? IconKeyboardDoubleArrowLeft : IconKeyboardDoubleArrowRight, {
       role: "button"
     }));
   }
